content/en/docs/tasks/observability/gateways/index.md

```diff
@@ -379,6 +379,21 @@ the [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/confi
     * Prometheus: `--set values.prometheus.enabled=true`
     * Tracing: `--set values.tracing.enabled=true`
 
+1. Setup the domain to expose addons. In this example, we will expose each addon on a subdomain, such as `grafana.example.com`.
+
+  * If you have an existing domain pointing to the external IP address of `istio-ingressgateway`:
+
+    {{< text bash >}}
+    $ export INGRESS_DOMAIN=<your.desired.domain>
+    {{< /text >}}
+
+  * If you do not have a domain, you may use [xip.io](xip.io) which will automatically resolve to the IP address provided
```

Comment:
We may want to switch to nip.io actually, seems to have better reviews. Knative is using xip.io and openshift uses nip.io out of the box so it adds some credibility to these. We can probably add a note that you should probably use a real domain for this and this is mostly for demo

Reply:
The insecure access bits were always meant to at least strongly imply demo /quick start usage.  So, adding a little disclaimer about that and the fact that Istio doesn't control nip or xip seems reasonable to me. Thanks for tackling this update!